docs/npm.md

```diff
@@ -0,0 +1,395 @@
+      "version": "1.0.0",
+      "license": "ISC",
+      "dependencies": {
+        "is-positive": "github:kevva/is-positive#75dd3a181375162eda014984f2decc663199b09e"
+      },
+      "devDependencies": {}
+    },
+    "foo/node_modules/is-positive": {
+      "version": "2.0.0",
+      "resolved": "git+ssh://git@github.com/kevva/is-positive.git#75dd3a181375162eda014984f2decc663199b09e",
+      "integrity": "sha512-q4goHhlrYSAw9hxTZwr4tcNV7jA9FydQdswnTc6B06EhqR7gsFJnvbcLw/fMAsF3HgYpoIA3BIxYRLtoVO6C1g==",
+      "license": "MIT",
+      "engines": {
+        "node": ">=0.10.0"
+      }
+    },
+    "node_modules/cachito-npm-without-deps": {
+      "version": "1.0.0",
+      "resolved": "git+ssh://git@github.com/cachito-testing/cachito-npm-without-deps.git#2f0ce1d7b1f8b35572d919428b965285a69583f6",
+      "dev": true
+    },
+    "node_modules/foo": {
+      "resolved": "foo",
+      "link": true
+    },
+    "node_modules/is-positive": {
+      "version": "1.0.0",
+      "resolved": "git+ssh://git@github.com/kevva/is-positive.git#1187a61f2e18cf7c11c23d61a1bd52b9fa6a5fdf",
+      "integrity": "sha512-+Si/uXLS2WcyBnOY1ZTP6KCxKmyxPnb90jjqJwVVcmGuM41yDKUcKwA4E2KhhjpvfTM0ocYXbkXtiCR0ah2rOA==",
+      "license": "MIT",
+      "engines": {
+        "node": ">=0.10.0"
+      }
+    }
+  }
+}
+```
+</details>
+
+<details>
+<summary>foo/package.json (workspace)</summary>
+
+```jsonc
+cat foo/package.json
+{
+  "name": "foo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+      "is-positive": "github:kevva/is-positive#75dd3a181375162eda014984f2decc663199b09e"
+  }
+}
+```
+</details>
+
+Cachi2 updates [required files](#required-files) for offline installation.
+
+Update root package.json file together with package.json files for each [workspace](https://docs.npmjs.com/cli/v9/using-npm/workspaces?v=true) with changes:
+* For git repositories and HTTPS URLs in dependencies update their value to an empty string
+
+Update the following in the package-lock.json file:
+* Delete integrity items in packages requested as git repositories. Cachi2 will clone the repository and will create a tar archive with its content. Integrity checksum will change and will no longer match the previous integrity checksum, which would result in error.
+* Replace URLs found in resolved items with local paths to [fetched dependencies](#using-fetched-dependencies). Applies only to root packages that are not local files or [workspaces](https://docs.npmjs.com/cli/v9/using-npm/workspaces?v=true).
+* Similarly to the above package.json changes, for git repositories and HTTPS URLs in package dependencies update their value to an empty string.
+
+If all these requirements were met, offline installation in the isolated container should be possible.
+
+#### Updated project example (for offline install):
```

Comment:
Since this whole part explains how cachi2 makes npm use the prefetched dependencies, maybe it would the perfect fit for the "Using fetched dependencies" section? :slightly_smiling_face:

Reply:
Moved in there